.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+<!-- What does this PR do? Please list any issue that these changes address and how you have gone about implementing the changes -->
+
+## Checklist
+
+I confirm that I have completed the following checks:
+
+**Testing**
+- [ ] If I have changed any of the Python or Fortran code, I have rebuilt and run the tests locally and they have passed.
+- [ ] I have addressed any differences in the regression tests caused by this pull request in the comments.
```

Comment:
I know what you mean, but this is rarely going to be easily explainable. Perhaps this could be changed to "justify large regression test changes" or similar? For discussion.